utils/dbus/Makefile

```diff
@@ -74,7 +74,6 @@ endef
 CMAKE_OPTIONS += \
 	-DDBUS_SYSTEM_SOCKET=/var/run/dbus/system_bus_socket \
 	-DDBUS_SESSION_SOCKET_DIR=/tmp \
-	-DDBUS_SYSTEM_PID_FILE=/var/run/dbus.pid \
```

Comment:
You sure about this?

Reply:
The cmake overview gives me the following output when building.
```
        System bus address:       unix:path=/var/run/dbus/system_bus_socket
        System bus socket:        /var/run/dbus/system_bus_socket
        System bus PID file:      /var/run/dbus/pid
        System bus user:          root
```

This is also the default pid location.
The procd writes its own pid file that is located under /var/run/dbus.pid
if I specify the one from the dbus in the procd, then the dbus service cannot start!

```
root@VR2-106149 /tmp/run/dbus # ls -la
drwxr-xr-x    2 root     root            80 Dec 10 14:16 .
drwxr-xr-x   11 root     root           620 Dec 10 14:05 ..
-rw-r--r--    1 root     root             6 Dec 10 14:16 pid
srwxrwxrwx    1 root     root             0 Dec 10 14:16 system_bus_socket
```

Reply:
hmm OK.